README.md

```diff
@@ -26,8 +26,11 @@ This project was originally implemented as a Vagrant deployment. If you are look
     - kube-workers (as many as you'd like, excluding the master)
     - kube-control (same as your master)
 2. Make any needed/custom modifications to the included `kube-deploy/group_vars/all.yml` configuration file.
-3. From the `kube-deploy` folder, use Ansible to deploy your cluster: `ansible-playbook -i ../inventory.ini kube-deploy.yml`
-
+3. From the root of the project:
```

Comment:
this makes the assumptions that some pre-requisites are installed prior to make and venv activation. can you outline in the docs what packages need to be installed for users of the project?